projects/1.18/assets/pyromancer/pyromancer/lang/zh_cn.json

```
@@ -1,226 +1,299 @@
-  "block.pyromancer.mercury_bricks_wall": "汞砖块墙",
-  "block.pyromancer.mysterious_vegetable": "奇怪的菜",
+  "block.pyromancer.infern": "狱火蕨",
+  "block.pyromancer.mercury_bricks_slab": "汞砖台阶",
+  "block.pyromancer.mercury_bricks_stairs": "汞砖楼梯",
+  "block.pyromancer.mercury_bricks_wall": "汞砖墙",
+  "block.pyromancer.mercury_bricks": "汞砖",
+  "block.pyromancer.mysterious_vegetable": "神秘蔬菜",
+  "block.pyromancer.nether_lily": "下界百合",
   "block.pyromancer.polished_frozen_mercury": "磨制冰制汞",
   "block.pyromancer.potted_faded_fungus": "褪色菌盆栽",
   "block.pyromancer.potted_faded_roots": "褪色菌索盆栽",
+  "block.pyromancer.pyromancer_table.tooltip.line1": "§e可用于为烈焰日志填充烈焰粉和更换翎笔",
   "block.pyromancer.pyromancer_table": "炽焰台",
-  "block.pyromancer.pyromancer_table.tooltip.line1": "§e可用于为烈焰日志填充烈焰粉并更换翎笔",
-  "block.pyromancer.pyrowood_door": "炽木木门",
+  "block.pyromancer.pyromoss_sprouts": "炽苔苗",
+  "block.pyromancer.pyromossed_netherrack": "炽苔下界岩",
+  "block.pyromancer.pyrowood_door": "炽木门",
   "block.pyromancer.pyrowood_fence": "炽木栅栏",
   "block.pyromancer.pyrowood_gate": "炽木栅栏门",
-  "block.pyromancer.pyrowood_leaves": "炽木树叶",
+  "block.pyromancer.pyrowood_leaves": "炽树树叶",
   "block.pyromancer.pyrowood_log": "炽木原木",
   "block.pyromancer.pyrowood_planks": "炽木木板",
-  "block.pyromancer.pyrowood_sapling": "炽木树苗",
+  "block.pyromancer.pyrowood_sapling": "炽树树苗",
   "block.pyromancer.pyrowood_slab": "炽木台阶",
   "block.pyromancer.pyrowood_stairs": "炽木楼梯",
   "block.pyromancer.pyrowood_trapdoor": "炽木活板门",
+  "block.pyromancer.rafflesia": "大王花",
+  "block.pyromancer.rotten_fungus": "朽腐菌",
+  "block.pyromancer.rotten_planks_door": "朽腐木门",
+  "block.pyromancer.rotten_planks_fence": "朽腐木栅栏",
+  "block.pyromancer.rotten_planks_gate": "朽腐木栅栏门",
+  "block.pyromancer.rotten_planks_slab": "朽腐木台阶",
+  "block.pyromancer.rotten_planks_stairs": "朽腐木楼梯",
+  "block.pyromancer.rotten_planks_trapdoor": "朽腐木活板门",
+  "block.pyromancer.rotten_planks": "朽腐木板",
+  "block.pyromancer.rotten_stem": "朽腐菌柄",
+  "block.pyromancer.rotten_wart_block": "朽腐疣块",
+  "block.pyromancer.scorch_nest": "灼炎虫巢穴",
+  "block.pyromancer.sizzling_vine": "酷热藤",
+  "block.pyromancer.stripped_pyrowood_log": "去皮炽木原木",
+  "block.pyromancer.stripped_rotten_stem": "去皮朽腐菌柄",
   "effect.brimflame": "硫火",
-  "effect.pigmen_legion_cd": "调试",
-  "effect.volcanic_mark_effect": "火山标记",
+  "effect.pigmen_legion_cd": "debug",
+  "effect.pyromancer.brimflame": "硫火",
+  "effect.pyromancer.cooling_coating_effect": "冷却覆层",
+  "effect.pyromancer.fiery_coating_effect": "火焰覆层",
+  "effect.pyromancer.incineration": "焚化",
+  "effect.pyromancer.mercury_coating_effect": "汞覆层",
+  "effect.pyromancer.pigmen_legion_cd": "debug",
+  "effect.pyromancer.rot_coating_effect": "朽腐覆层",
+  "effect.pyromancer.rot": "朽腐",
+  "effect.pyromancer.volcanic_mark_effect": "火山印记",
+  "effect.volcanic_mark_effect": "火山印记",
+  "enchantment.pyromancer.ashen_pages.desc": "烈焰日志不再能在炽焰台处充能，而是在使用其击杀目标时充能。",
   "enchantment.pyromancer.ashen_pages": "书页付灰",
```

| **审查级别** |
|--------------|
| ℹ️ 信息 / 不阻止合并  |
```suggestion
  "enchantment.pyromancer.ashen_pages": "书页付灰",
```

没看懂

原文：**Ashen Pages**

---

这个魔咒适用于烈焰日志，参见`item.pyromancer.blazing_journal.tooltip.line1`这些键

觉得名称大概还行就没改